src/pint/scripts/export_clocks.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import os
+import sys
+
+import pint.logging
+from loguru import logger as log
+
+pint.logging.setup(level=pint.logging.script_level)
+
+import pint.observatory
+import pint.observatory.topo_obs
+
+__all__ = ["main"]
+
+
+def main(argv=None):
+    parser = argparse.ArgumentParser(
+        description="PINT tool for export clock files",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        epilog=f"Example: export_clocks -o clock_dir/ GBT Arecibo CHIME VLA",
+    )
+    parser.add_argument(
+        "observatories", help="Observatories to export", type=str, nargs="+"
+    )
+    parser.add_argument(
+        "-o",
+        "--out",
+        default=os.path.abspath(os.curdir),
+        type=str,
+        dest="directory",
+        help="Destination directory (set $PINT_CLOCK_OVERRIDE to this value to look here first for clock files)",
+    )
+    parser.add_argument(
```

Comment:
More conventional would be a `--verbose` (possibly repeatable) and optionally a `--quiet` rather than explicitly referencing log levels. Since this is a specific application, one might be able to select sensible log levels to correspond to these.

Reply:
That's the type of interface I had gone for initially for applications, but @scottransom wanted this method

Reply:
And if the discussion is about the log levels, I was just thinking that we should be consistent with the other PINT command line programs.  Maybe `--verbose` could correspond to upping the log level by one, as well. (maybe by one for each verbose added).

Reply:
Yes, this is about log levels. I'm not sure users of a command line program will have any idea what log levels have to do with what output they want to see. Making `--verbose` bump up the log level, and `--quiet` bump it down by one might be a tidy way to present a more conventional command-line interface.

Reply:
Yes, this is what I had implemented months ago.  I agree it's more standard but I don't really care either way.

Reply:
See #1377 for discussion of this.